modyn/tests/selector/internal/selector_strategies/test_actual_samplers.py

```diff
@@ -5,6 +5,7 @@
 
```

Comment:
One small thing: Could you update the file structure in the tests such that we have one test_blabla.py per blabla.py in the main source? So instead of test_actual_samplers, we have the tests in the respective test_loss_downsampling_strategy.py etc. I know the tests are short but otherwise it is hard to navigate what is tested where